### PR TITLE
feat: add view templates option to dashboard menu

### PR DESCRIPTION
**Pull request by sharpshooter90**
### Summary

Added "View templates" CTA to add new dashboard flow. Currently there is no way to explore the template in though our app. This will help the users to discover the our dashboard templates quickly while they are in need.

#### Related Issues / PR's
https://github.com/SigNoz/engineering-pod/issues/1451

#### Screenshots
image
image

### CODE REVIEW

File: frontend/src/container/ListOfDashboard/DashboardsList.tsx [hunk 1]
```diff
@@ -600,6 +604,19 @@ function DashboardsList(): JSX.Element {
 				),
 				key: '1',
 			},
+			{
+				label: (
+					<a
+						className="create-dashboard-menu-item"
```
Comment (vikrantgupta25) [inline frontend/src/container/ListOfDashboard/DashboardsList.tsx:610]: does this work in the light mode correctly ? we do not need any changes for the same ?

**Comment by GeekBoySupreme** [comment]
Can we make the dropdown wider? The arrow icon is too close to the text. A better way to nudge users towards dashboard templates might have been like this -
image

File: frontend/src/container/ListOfDashboard/DashboardsList.tsx (hunk 1 above)
Comment (sharpshooter90) [reply frontend/src/container/ListOfDashboard/DashboardsList.tsx:610]: Screenshot 2024-08-16 at 11 52 59 AM
Screenshot 2024-08-16 at 11 53 13 AM
Sharing the light theme. @vikrantgupta25 did you see any issues from the code perspective or visually?

**Comment by sharpshooter90** [comment]
> Can we make the dropdown wider? The arrow icon is too close to the text. A better way to nudge users towards dashboard templates might have been like this - image

@GeekBoySupreme I scoped this as a quick change, let us plan for a better template view native to the app in later cycles.  I have fixed the width to 200px so that the dropdown.button will have some breathing space.

**Review by YounixM — APPROVED** [review]

**Review by vikrantgupta25 — APPROVED** [review]

**Pull request merged**